Autopilot/PathManager/Inc/pathDatatypes.hpp

```diff
@@ -29,6 +30,8 @@ struct _PassbyControl{
 
     double throttlePercent;
     bool throttlePassby;
+    _PassbyControl() : rollPercent(0), rollPassby(false), rudderPercent(0), rudderPassby(false), pitchPercent(0), pitchPassby(false), throttlePercent(0), throttlePassby(false)
```

Comment:
My rule of thumb:

structs should only be used for plain old data, that is, data only and no methods. If you are adding a constructor I would make this a class.